README.md

```diff
@@ -58,11 +58,6 @@ To get started with the project, follow these steps:
 This project is licensed under the MIT License - see the [LICENSE](LICENSE) file for details.
 
 ## Contributors 
-
```

Comment:
Add this too. And count badge before the list

Reply:
The problem might be on contrib.rocks' end. It could be an issue with their API fetching process or a bug in how they pull the contributors. It is an internal server error. See the github api is fetching all the contributors correctly but the contrib.rocks is not fetching all the recent contributors which could be their internal server error as the code is perfect. So its better to only keep the count badge and at the bottom right we can also see the contributors too. Thank you!
![contributor](https://github.com/user-attachments/assets/47f04482-88e4-4554-9e73-30b8d6f5e1fe)